R/helpers.R

```diff
@@ -34,22 +29,9 @@
     commands <- c(downloadCommand)
   }
 
-  envFile <- paste0(taskId, ".rds")
```

Comment:
I think I might be missing something here, but don't we still need to upload a single argument blob to storage? Is that what was indended with the azbatchenv environment in the job prep? Are there any cases where this object gets too big  will not work? Would we ever need to chunk it up?